deploy.sh

```
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+echo "🟡 Running Terraform..."
+cd terraform
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)